src/ui/zcl_abapgit_password_dialog.clas.abap

```diff
@@ -22,10 +22,39 @@ CLASS ZCL_ABAPGIT_PASSWORD_DIALOG IMPLEMENTATION.
 
   METHOD popup.
 
-    PERFORM password_popup
-      IN PROGRAM (sy-cprog)
-      USING iv_repo_url
-      CHANGING cv_user cv_pass.
+    DATA lv_gui_is_available TYPE abap_bool.
```

Comment:
suggest moving this part to a different pull request

Reply:
I‘ll try to remove this class form the current PR and create a new PR for this 'intermediate' solution soon...